### PR TITLE
V3.5.1 fbx editing template add smart material convesion property

### PR DESCRIPTION
**Pull request by arsen2010**
Re: # https://github.com/cocos/3d-tasks/issues/12294

### Changelog

* 

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check // Manual trigger with `@cocos-robot run test cases` afterward

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: editor/i18n/en/assets.js [hunk 1]
```diff
@@ -318,10 +318,16 @@ module.exports = {
...
                 title:
-                    'When exporting FBX animations, whether prefer to use the time range recorded in FBX file. ' +
-                    'If one is not preferred, or one is invalid for use, the time range is robustly calculated. ' +
+                    'When exporting FBX animations, whether prefer to use the time range recorded in FBX file.<br>' +
+                    'If one is not preferred, or one is invalid for use, the time range is robustly calculated.<br>' +
                     'Some FBX generators may not export this information.',
             },
+            smartMaterialEnabled: {
+                name: 'Smart Material Conversion',
+                title: 'Convert DCC material to engine material, and match the lighting model of DCC material.',
```
Comment (pandamicro) [inline editor/i18n/en/assets.js]: Convert DCC materials to engine builtin materials which match the internal lighting model.

File: editor/i18n/en/assets.js [hunk 2]
```diff
@@ -318,10 +318,16 @@ module.exports = {
...
+                warn: 'For the Smart Material Conversion of laboratory function in preferences is turned off, ' +
```
Comment (pandamicro) [inline editor/i18n/en/assets.js]: Should forbid changing this option when laboratory feature is disabled

**Review by pandamicro — CHANGES_REQUESTED** [review]

File: editor/i18n/en/assets.js (hunk 2 above)
Comment (pandamicro) [reply editor/i18n/en/assets.js]: warning text only showing when laboratory feature is disabled:

"The Smart Material Conversion feature have been disabled in laboratory preference, please enable it to change model level setting."

File: editor/i18n/zh/assets.js [hunk 3]
```diff
@@ -312,7 +312,12 @@ module.exports = {
             },
             preferLocalTimeSpan: {
                 name: '优先使用文件时间范围',
-                title: '在导出 FBX 动画时，是否优先使用 FBX 文件中记录的动画时间范围。若不使用该时间范围或此范围不可能用，则会粗略地计算动画时间范围。有些 FBX 生产工具中可能并未导出该信息。',
+                title: '在导出 FBX 动画时，是否优先使用 FBX 文件中记录的动画时间范围。<br>若不使用该时间范围或此范围不可能用，则会粗略地计算动画时间范围。有些 FBX 生产工具中可能并未导出该信息。',
+            },
+            smartMaterialEnabled: {
+                name: '材质智能转换',
+                title: '将 DCC 材质转化为引擎材质, 并匹对 DCC 材质的光照模型',
+                warn: '由于偏好设置里的实验室功能 "材质智能转换" 已关闭，本属性的预期值为不勾选，与现值勾选不匹配。',
```
Comment (pandamicro) [inline editor/i18n/zh/assets.js]: 只应该在实验室功能关闭时显示：
"偏好设置里的实验室功能 "材质智能转换" 已关闭，请启用此项功能来修改模型级别设置。"

File: editor/inspector/assets/fbx/fbx.js [hunk 4]
```diff
@@ -154,9 +176,36 @@ const Elements = {
...
+    smartMaterialEnabled: {
+        ready() {
+            const panel = this;
+
+            panel.$.smartMaterialEnabledCheckbox.addEventListener('change', panel.setProp.bind(panel, 'fbx.smartMaterialEnabled', 'boolean'));
+        },
+        async update() {
+            const panel = this;
+
+            const laboratoryExpectedValue = await Editor.Profile.getConfig('inspector', 'asset.fbx.material.smart');
```
Comment (VisualSJ) [inline editor/inspector/assets/fbx/fbx.js]: engine~
```javascript
await Editor.Profile.getConfig('engine', 'asset.fbx.material.smart');
```

**Review by pandamicro — APPROVED** [review]

**Review by VisualSJ — APPROVED** [review]

**Pull request merged**